### PR TITLE
Fix: Showing result based on query after refresh

### LINKED ISSUE #1218: Bug while refreshing layout
**Actual Behaviour**

Search something in tracks fragment. It will look something like this.

screenshot_20170303-050132

Now refresh the layout by swiping down. It becomes like this. All the tracks are shown irrespective of the search query.

screenshot_20170303-044622

**Expected Behaviour**

It should be same as the first picture or else if after refreshing, new tracks are added to the database, only those containing "ai" should be displayed. 

**Would you like to work on the issue?**
yes

### PR DESCRIPTION
**Pull request by chiragw15**
Fixes #1218 
Changes: Now when refreshed, only those items which matches with search query are displayed.

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/fragments/DayScheduleFragment.java [hunk 1]
```diff
@@ -231,9 +231,13 @@ public void onScheduleDownloadDone(SessionDownloadEvent event) {
 
         swipeRefreshLayout.setRefreshing(false);
         if (event.isState()) {
-            if (dayScheduleAdapter != null)
-                dayScheduleAdapter.refresh();
-
+            if (dayScheduleAdapter != null && searchView!=null){
```
Comment (harshitagupta30) [inline android/app/src/main/java/org/fossasia/openevent/fragments/DayScheduleFragment.java]: Reformat the code.

File: android/app/src/main/java/org/fossasia/openevent/fragments/DayScheduleFragment.java [hunk 2]
```diff
@@ -47,16 +47,13 @@
...
+    TextView noSchedule;
     private String searchText = "";
-
     private SearchView searchView;
-
     private SwipeRefreshLayout swipeRefreshLayout;
-
-    @BindView(R.id.list_schedule) RecyclerView dayRecyclerView;
-    @BindView(R.id.txt_no_schedule) TextView noSchedule;
-
```
Comment (mananwason) [inline android/app/src/main/java/org/fossasia/openevent/fragments/DayScheduleFragment.java]: Please refrain from such refactoring in your code. It's impossible to review this way

**Comment by chiragw15** [comment]
@mananwason corrected here too. Sorry for trouble.

**Comment by mananwason** [comment]
LGTM

**Pull request merged**